gcsweb/cmd/gcsweb/gcsweb_test.go

```diff
@@ -209,7 +209,7 @@ func TestHandleObject(t *testing.T) {
 
 			s := server{storageClient: client}
 
-			err = s.handleObject(w, tc.bucket, tc.object)
+			err = s.handleObject(w, tc.bucket, tc.object, objectHeaders{})
```

Comment:
Ideally we would check that headers are set  correctly, too, in the test

Reply:
Yea that will need to actually have different file content and check the desired headers as well.

Reply:
We will need to change the mock server also to generate the headers as well. I will need to study their docs etc. This sounds like a nice follow-up

Reply:
addressed